backend/api/team.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, Depends
+from ..models.team import Team
+from ..services.team import TeamService
+
+__authors__ = ["Mustafa Aljumayli"]
+
+api = APIRouter(prefix="/api/test/teams")
+
+openapi_tags = {
+    "name": "Teams",
+    "description": "These routes are only used for demo/setup confirmation purposes.",
+}
+
+
+@api.get("/", tags=["Teams"])
+async def get_teams(team_service: TeamService = Depends()):
+    """Gets all teams"""
+    return team_service.get_all_teams()
+
+
+@api.get("/id/{id}", tags=["Teams"])
+async def get_team(id: int, team_service: TeamService = Depends()) -> Team:
```

Comment:
Question: I see these are currently just test routes, but we should probably have a function that returns the team object of the currently logged in user. How would we go about reading the JWT that is sent and retrieving that team object?

Reply:
I can create this, but just need the info!

Reply:
Returning the team object might compromise the safety of the passwords. Instead, I encoded TokenData and created a route to decode that data back when the user is actively logged in🫡.